packages/components/nodes/agents/OpenAIFunctionAgent/OpenAIFunctionAgent.ts

```diff
@@ -67,6 +77,19 @@ class OpenAIFunctionAgent_Agents implements INode {
 
     async run(nodeData: INodeData, input: string, options: ICommonObject): Promise<string | ICommonObject> {
         const memory = nodeData.inputs?.memory as FlowiseMemory
+        const moderations = nodeData.inputs?.inputModeration as Moderation[]
+
+        if (moderations && moderations.length > 0) {
+            try {
+                // Use the output of the moderation chain as input for the OpenAI Function Agent
+                input = await checkInputs(moderations, input)
+            } catch (e) {
+                await new Promise((resolve) => setTimeout(resolve, 500))
+                //streamResponse(options.socketIO && options.socketIOClientId, e.message, options.socketIO, options.socketIOClientId)
```

Comment:
@ocflowiseai we should enable the `streamResponse` for some of the agents/chains.

For full list of which chains/agents are available for streaming, refer [here](https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/utils/index.ts#L915)

Reply:
Sure thing! I will open a separate PR for that.